style.css

```diff
@@ -1,3 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&display=swap');
+
 * {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+* input {
+  margin: 0;
+  padding: 0;
+  border-radius: 10px;
+}
+
+body {
+  position: relative;
+  height: 95vh;
+  color: black;
+  font-size: 16px;
+  font-family: 'Architects Daughter', cursive;
+  background-image: url('images/bg.jpg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: auto;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 16px;
+  margin-bottom: 16px;
+}
+
+h1,
+h2 {
+  text-align: center;
+}
+
+.book-list {
+  width: 100%;
+  border: 2px solid rgb(39, 37, 37);
+  border-radius: 3px;
+  margin-top: 15px;
+  list-style: none;
+}
+
+.book-list li {
```

Comment:
- [ ] OPTIONAL
Kindly add some **spacing** between the _book title_ and the _book author_.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/138251223-4dd83be6-bd3a-4cc0-9f0c-3bead8c66d29.png)

![image](https://user-images.githubusercontent.com/12745474/138251384-ad113242-76ee-4b25-88e0-83f49805eab6.png)

Reply:
Thanks for your review. We'll fix that problem